src/dotnet-uninstall/Messages.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Microsoft.DotNet.Tools.Uninstall
+{
+    internal static class Messages
```

Comment:
Should this be localizablestrings.resx, preparing it for localization? Maybe file an issue do make the localizablestrings change later on, given this is already a big PR.

Reply:
Yes, @wli3 said that he would add the localization infrastructure. It's already in Issue #7.